.github/workflows/frontend_ci.yml

```diff
@@ -0,0 +1,33 @@
+name: React CI
+
+on:
+  push:
+    branches: [main, staging, production]
+  pull_request:
+    branches: [main, staging, production]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Usar Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Instalar dependencias
+      run: npm ci
```

Comment:
Los comandos no van a funcionar porque se van a ejecutar en la carpeta raíz del repositorio.

Leer: https://stackoverflow.com/questions/58139175/running-actions-in-another-directory